views/animelist.ejs

```diff
@@ -14,7 +14,7 @@
     <div class="page-single">
             <div class="container">
                 <div class="row ipad-width2">
-                    <div class="col-md-8 col-sm-12 col-xs-12">
+                    <div class="col-md-8 col-sm-12 col-xs-12" style="width: 100%">
```

Comment:
This is just a small change in css of the anime list columns. They were pushed to the left hand side of the screen previously, as mentioned in issue #12.